scipp-imaging-2D.ipynb

```diff
@@ -0,0 +1,8350 @@
+      "GP2_007-003-WFM-open-beam_0813.tiff: Image 814, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0814.tiff: Image 815, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0815.tiff: Image 816, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0816.tiff: Image 817, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0817.tiff: Image 818, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0818.tiff: Image 819, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0819.tiff: Image 820, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0820.tiff: Image 821, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0821.tiff: Image 822, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0822.tiff: Image 823, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0823.tiff: Image 824, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0824.tiff: Image 825, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0825.tiff: Image 826, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0826.tiff: Image 827, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0827.tiff: Image 828, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0828.tiff: Image 829, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0829.tiff: Image 830, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0830.tiff: Image 831, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0831.tiff: Image 832, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0832.tiff: Image 833, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0833.tiff: Image 834, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0834.tiff: Image 835, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0835.tiff: Image 836, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0836.tiff: Image 837, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0837.tiff: Image 838, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0838.tiff: Image 839, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0839.tiff: Image 840, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0840.tiff: Image 841, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0841.tiff: Image 842, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0842.tiff: Image 843, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0843.tiff: Image 844, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0844.tiff: Image 845, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0845.tiff: Image 846, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0846.tiff: Image 847, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0847.tiff: Image 848, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0848.tiff: Image 849, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0849.tiff: Image 850, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0850.tiff: Image 851, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0851.tiff: Image 852, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0852.tiff: Image 853, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0853.tiff: Image 854, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0854.tiff: Image 855, of 2001\r",
+      "GP2_007-003-WFM-open-beam_0855.tiff: Image 856, of 2001\r",
+      "GP2_007
```

Comment:
Was it a conscious decision not to use the functions from `compat/mantid`?

Reply:
Good catch, `load_component_info_from_instrument_file` didn't exist (although the others did but I copied them for ease). But it is in the scipp PR now so I'll remove this block